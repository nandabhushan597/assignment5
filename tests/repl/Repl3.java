:{
  3
    :}
:{
  /* nothing to see here */
  :}
:{
  /* unterminated comment
:}
:{ // comment on first line
// more commentary
34f
:}
:{ // comment :}
true
:}
