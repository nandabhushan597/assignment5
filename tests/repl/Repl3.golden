Exit code: ExitSuccess
Stdout:
Welcome to the Java interpreter.
Type `:?` at the prompt for the list of commands.
Java> |> |> 3
Java> |> |> Java> |> |> Parsing error when list of remaining tokens starts with:
[OperatorT DivideO,OperatorT TimesO,IdentifierT "unterminated",IdentifierT "comment"]
Java> |> |> |> 34.0f
Java> |> |> true
Java> Good-bye.

Stderr:
